addons/interaction/functions/fnc_modifyJoinGroupAction.sqf

```diff
@@ -0,0 +1,26 @@
+/*
+ * Author: PabstMirror
+ * Modifies the ACE_JoinGroup action to show group name.
+ *
+ * Arguments:
+ * 0: Target <OBJECT>
+ * 1: Player <OBJECT>
+ * 2: Args <Any>
+ * 3: Action Data <ARRAY>
+ *
+ * Return Value:
+ * None
+ *
+ * Example:
+ * [cursorObject, player, [], []] call ace_interaction_fnc_modifyJoinGroupAction;
```

Comment:
No semi-colon at the end (I think that's what we decided to do... but no one put it on the wiki yet?).